integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/Firmware.feature

```diff
@@ -153,15 +150,12 @@ Feature: SmartMetering Configuration - Firmware
       | DeviceIdentification      | TEST1024000000002 |
       | FirmwareModuleVersionFunc | M57 4836          |
     Then the update firmware result should be returned
-      | DeviceIdentification      | TEST1024000000002      |
-      | FirmwareModuleVersionComm | Telit 10.00.154        |
-      | FirmwareModuleVersionMa   | BL_012 XMX_N42_GprsV09 |
-      | FirmwareModuleVersionFunc | M57 4836               |
-    And the database should be updated with the new device firmware
-      | DeviceIdentification      | TEST1024000000002      |
-      | FirmwareModuleVersionComm | Telit 10.00.154        |
-      | FirmwareModuleVersionMa   | BL_012 XMX_N42_GprsV09 |
-      | FirmwareModuleVersionFunc | M57 4836               |
+      | DeviceIdentification      | TEST1024000000002 |
+    And the database should not be updated with the new device firmware
+      | DeviceIdentification      | TEST1024000000002 |
+      | FirmwareModuleVersionComm |                   |
+      | FirmwareModuleVersionMa   |                   |
+      | FirmwareModuleVersionFunc |                   |
```

Comment:
Same question here.

Reply:
Yes, it was empty and it should stay empty